### PR TITLE
Add SECURITY.md

### PR DESCRIPTION
**Pull request by bryanculver**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Nautobot Security Policy
+
+## Security Announcements
+
+Join the #nautobot channel in the Network to Code Slack for messages about security and release announcements.
+
+## Report a Vulnerability
+
+We're appreciate the time security researchers and users contribute to reporting vulnerabilities to the Nautobot Community.
```
Comment (glennmatthews) [inline SECURITY.md:9]:
```suggestion
We appreciate the time security researchers and users contribute to reporting vulnerabilities to the Nautobot Community.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+If you feel your report is safe for public disclosure (a CVE related to a dependency, or a low-risk bug) please feel free to open a bug [issue on GitHub](https://github.com/nautobot/nautobot/issues/new/choose). 
+
+If you are unsure of the severity of your report or you feel it should not be publicly disclosed until a fix has been release, you can also email [security@networktocode.com](mailto:security@networktocode.com) with the security details.
```
Comment (glennmatthews) [inline SECURITY.md:13]:
```suggestion
If you are unsure of the severity of your report or you feel it should not be publicly disclosed until a fix has been released, you can also email [security@networktocode.com](mailto:security@networktocode.com) with the security details.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+- You need help configuring Nautobot security settings (such as external auth)
+- You need help applying security related updates
+- Your issue is not security related
+
+## Security Vulnerability Response
+
+Each report is acknowledged and analyzed by security response members within five (5) working days.
+
+Any vulnerability information shared with security response members stays within Nautobot project and will not be disseminated to other projects unless it is necessary to get the issue fixed.
```
Comment (glennmatthews) [inline SECURITY.md:33]:
```suggestion
Any vulnerability information shared with security response members stays within the Nautobot project and will not be disseminated to other projects unless it is necessary to get the issue fixed.
```

**Review by glennmatthews — COMMENTED** [review]
Just a few quick nitpicks - thanks for starting this!

**Comment by bryanculver** [comment]
Thanks for the quick feedback @glennmatthews!

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+## Public Disclosure Timing
+
+A public disclosure date is negotiated by the Nautobot maintainers and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation is available. It is reasonable to delay disclosure when the bug or the fix is not yet fully understood, the solution is not well-tested, or for vendor coordination. The timeframe for disclosure is from immediate (especially if it's already publicly known) to a few weeks. For a vulnerability with a straightforward mitigation, we expect report date to disclosure date to be on the order of ten (10) days. The Nautobot maintainers hold the final say when setting a disclosure date.
+
+## Team 
+
+Below are the current team members responsible for receiving and triaging Nautobot security issues.
+
+- Jathan McCollum (**[@jathanism](https://github.com/jathanism)**) `<jathan.mccollum@networktocode.com>` [[4096R/9B1EC9E6D0B1B49F64C2F97AA539DE2F671694A8]](https://keybase.io/jathanism/pgp_keys.asc)
+- Glenn Matthews (**[@glennmatthews](https://github.com/glennmatthews)**) `<glenn.matthews@networktocode.com>`
```
Comment (glennmatthews) [inline SECURITY.md:46]:
```suggestion
- Glenn Matthews (**[@glennmatthews](https://github.com/glennmatthews)**) `<glenn.matthews@networktocode.com>` [[4096R/C3DF1C5D9727F82ACF8F743238BF0D0E68B9F76C]](https://keybase.io/glennmatthews/pgp_keys.asc)
```

**Pull request merged**